_includes/manuals/nightly/3.5.2_configuration_options.md

```diff
@@ -363,7 +363,7 @@ Default: 35
 
 ##### puppet_server
```

Comment:
My bad, but I removed this setting in 1.24 and forgot it here (https://github.com/theforeman/foreman/commit/e7bc38eaaca928503232a07f2e564fc462aeb02c).

Just a thought: can we store these config options and their descriptions in foreman.git and have a script that exports them for the manual?